galaxy_ng/app/api/ui/viewsets/group.py

```diff
@@ -28,6 +28,7 @@ class GroupViewSet(LocalSettingsMixin, viewsets.GroupViewSet):
     filter_backends = (DjangoFilterBackend,)
     filterset_class = GroupFilter
     permission_classes = [access_policy.GroupAccessPolicy]
+    queryset = Group.objects
```

Comment:
[requires response]

Not 'Group.objects.all()' ?  

If so, kind of curious what the difference is.

Reply:
I'm actually not sure what the difference is. I noticed `Model.objects` was used in pulpcore. For consistency I'll put the all() back